src/common/components/Icons/CheckIcon.tsx

```diff
@@ -17,6 +19,9 @@ export const CheckIcon = ({
       fill='none'
       xmlns='http://www.w3.org/2000/svg'
       className={className}
+      onClick={() => {
+        onClick && onClick()
+      }}
```

Comment:
NB, just fyi: Could be simplified to `onClick={onClick}` here and in other icons, as it's fine to pass `undefined` to `onClick`

Reply:
fixed